XenAdmin/Controls/GPU/GpuConfiguration.cs

```diff
@@ -153,6 +155,22 @@ private void SetWindowTitle()
                                    ? Messages.GPU_RUBRIC_PLEASE_SELECT_WHICH_GPU_ONE
                                    : Messages.GPU_RUBRIC_PLEASE_SELECT_WHICH_GPU_MULTIPLE;
         }
+
+        private void HideColumnIfEmpty(string columnName)

```

Comment:
I would change this method to accept as a parameter 'DataGridViewColumn column' and then in line 164 you can do row.Cells[column.Name] to find the cell. In general using hardcoded strings should be avoided, because if someone renames the columns, lines 105-106 will break.